use-cases/data-export.md

```diff
@@ -19,19 +19,19 @@ The _Data export_ feature of the **Mews Connector API** provides a streamlined m
 
 > ### Restricted!
 >
-> This feature is currently under beta test and is subject to change. If you would like to take advantage of the feature, contact us via [partnersuccess@mews.com](mailto://partnersuccess@mews.com).
+> This feature is currently under beta test and is subject to change. If you would like to take advantage of the feature, contact us via [partnersuccess@mews.com](mailto:partnersuccess@mews.com).
 
 ## Initiate the export
 
-To initiate a bulk data export, use the [Add export](../operations/exports.md#add-export) operation, specifying the data _Entity type_, e.g. `OrderItem`.
+To initiate a bulk data export, use the [Add export](../operations/exports.md#add-export) operation, specifying the data _Entity type_, e.g. `OrderItem`. The operation returns an [Export](../operations/exports.md#export) object containing the Export `Id` to use in the next step. If there is already an export pending, the operation returns a 400 HTTP error code.
 
 | <div style="width:350px">'How to' use case</div> | API Operations |
 | :-- | :-- |
 | How to create a new bulk data export | [Add export](../operations/exports.md#add-export) |
 
 ## Check export status
 
-To monitor the progress of the export and retrieve the exported data, use the [Get all exports](../operations/exports.md#get-all-exports) operation. This operation provides a list of all ongoing and completed exports, including their status.
+To monitor the progress of the export and retrieve the exported data, use the [Get all exports](../operations/exports.md#get-all-exports) operation. This operation returns details of exports for the `Id`s you specify, including their status. You can repeatedly check the status of an export while its status is `Pending` or `Processing`, for example every 5 minutes.
```

Comment:
suggestion(non-blocking): Perhaps this will create more confusion, but the retrieved `Id` is passed in `ExportIds` array. 😬 

```suggestion
To monitor the progress of the export and retrieve the exported data, use the [Get all exports](../operations/exports.md#get-all-exports) operation. This operation returns details of exports for the `ExportIds` you specify, including their status. You can repeatedly check the status of an export while its status is `Pending` or `Processing`, for example every 5 minutes.
```